Content.Server/EntityEffects/Effects/PlantMetabolism/PlantAdjustPotency.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Content.Server.Botany.Systems;
+using Content.Shared.EntityEffects;
+
+namespace Content.Server.EntityEffects.Effects.PlantMetabolism;
+
+public sealed partial class PlantAdjustPotency : PlantAdjustAttribute
+{
+    /// <summary>
+    ///     Handles increase or decrease of plant potency.
+    /// </summary>
```

Comment:
Move the three summary lines above `public sealed partial class PlantAdjustPotency : PlantAdjustAttribute` because that is what you are writing the summary about.